Octokit.Tests/Clients/RepositoryPagesClientTests.cs

```diff
@@ -50,8 +50,9 @@ public void RequestsCorrectUrl()
                 var connection = Substitute.For<IApiConnection>();
                 var client = new RepositoryPagesClient(connection);
 
-                await Assert.ThrowsAsync<ArgumentNullException>(() => client.Get(null, "name"));
-                await Assert.ThrowsAsync<ArgumentNullException>(() => client.Get("owner", null));
+                await Assert.ThrowsAsync<ArgumentNullException>(() => client.GetAll(null, "name", new ApiOptions()));
+                await Assert.ThrowsAsync<ArgumentNullException>(() => client.GetAll("owner", null, new ApiOptions()));
+                await Assert.ThrowsAsync<ArgumentNullException>(() => client.GetAll("owner", "name", null));
             }
```

Comment:
Can you also add a `EnsuresNonEmptyArguments()` test function that passes `""` into the variuos parameters and ensures exceptions are thrown?